packages/treat/src/createContentHash.ts

```diff
@@ -1,10 +1,8 @@
-import crypto from 'crypto';
+import objectHash from 'object-hash';
 
 export const createContentHash = (content: any) => {
-  return crypto
-    .createHash('sha1')
-    .update(JSON.stringify(content))
-    .digest('base64')
+  return objectHash
+    .sha1(content)
     .slice(0, 5)
-    .replace(/^((-?[0-9])|--)/, '_$1');
-};
+    .replace(/(^[0-9])/, '_$1');
```

Comment:
Can you explain this change for us?

Reply:
I've simplified the regex as the SHA1 hash will never produce the hyphens represented in the original pattern thus making those additional clauses redundant. We only need to ensure that the output doesn't start with a digit, if it does, it will be prefixed with an underscore to transform it into a valid CSS identifier.